web/src/api/fbaCalcAPI.ts

```diff
@@ -59,6 +59,11 @@ export async function postFBAStations(
 ): Promise<FBAWeatherStationsResponse> {
   const url = '/fba-calc/stations'
 
+  // Filter out bad data.
+  fireBehaviorStations = fireBehaviorStations.filter(
+    station => !isNaN(station.stationCode)
```

Comment:
So does WFWX unset station codes for stations that are deleted?

Reply:
No. WFWX stops giving us the station, somewhere in firebat the mapping from station code in url, to station from the list fails. That would probably be a better place to fix it, just not adding the row.